### PR TITLE
Feature/28 doors and levels game

### PR DESCRIPTION
**Pull request by MihailRom**
This is game, where user can select any number include number 0. If user select non zero number - game should plot "next level" numbers. If user select zero value, console application should go to upper level.
Issue #28

### CODE REVIEW

File: HW1/Mikhail_Romashov/DoorsAndLevels/DoorsAndLevels/DoorsAndLevels.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            {
+                throw new Exception("Number is not in list! Choose again.");
+            }
+
+
+            if (coeff == 0)
+            {
+                if (m_levelCoeff.Count == 0)  //stack is empty
+                {
+                    throw new Exception("It is first level! Choose again.");
```
Comment (gNage) [inline HW1/Mikhail_Romashov/DoorsAndLevels/DoorsAndLevels/DoorsAndLevels.cs]: really weird way to return from method but it works

Comment (MihailRom) [reply HW1/Mikhail_Romashov/DoorsAndLevels/DoorsAndLevels/DoorsAndLevels.cs]: Is it better to use "return" with some value?

Comment (gNage) [reply HW1/Mikhail_Romashov/DoorsAndLevels/DoorsAndLevels/DoorsAndLevels.cs]: I didn't find some proof why you shouldn't throw exeptions like that, but i think if you used some method you wouldn't expect thay it throw exeption instead just return string.
Now it works, fine for tactical realization. I believe you should fix it in next stage

File: HW1/Mikhail_Romashov/DoorsAndLevels/DoorsAndLevels/DoorsAndLevels.cs [hunk 2]
```diff
@@ -61,22 +62,33 @@ public void CalcLevel(int coeff)
...
         {
-            m_arrayNum[0] = 1;
-            m_arrayNum[1] = 2;
-            m_arrayNum[2] = 3;
-            m_arrayNum[3] = 4;
+            var random = new Random();
+            int sizeLst = 9;
+            List<int> lstWithNum = new List<int>(sizeLst);
+            for (int i = 0; i < sizeLst; i++) //Add to List values 1..9
+                lstWithNum.Add(i+1);
```
Comment (gNage) [inline HW1/Mikhail_Romashov/DoorsAndLevels/DoorsAndLevels/DoorsAndLevels.cs]: For this `lstWithNum` list would be nice use [Enumerable.Range](https://docs.microsoft.com/ru-ru/dotnet/api/system.linq.enumerable.range?view=netframework-4.8) method

**Pull request merged**